sharing_portal/templates/sharing_portal/edit.html

```diff
@@ -70,6 +76,20 @@ <h4><i class="fa fa-files-o"></i> Versions</h4>
                 event.preventDefault();
               }
             });
+          })
+          const deleteAllButton = document.querySelector('[name=delete_all]');
+          deleteAllButton.addEventListener('click', (event) => {
+            let warning = "WARNING! You are about to delete your entire Trovi artifact! " +
+              "Are you sure you want to do this? Type the name of the artifact to confirm:";
+            while (true) {
```

Comment:
I don't think you need a while: true here; the prompt should be blocking. In fact, won't this mean the thing keeps coming up if the user tries to cancel via another mechanism?